scripts/run-fvt.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+SCRIPT_PATH=$PWD/$(dirname "$0")
+KIND_CLUSTER_NAME="staticroute-operator-fvt"
+DEBUG="${DEBUG:-false}"
+# shellcheck source=scripts/fvt-tools.sh
+. "${SCRIPT_PATH}/fvt-tools.sh"
+
+cleanup() {
+  fvtlog "Running cleanup, error code $?"
+  if [[ "${DEBUG}" == "false" ]]; then
+    kind delete cluster --name ${KIND_CLUSTER_NAME}
+  fi
+}
+
+trap cleanup EXIT
+
+fvtlog "Starting staticroute-operator fvt testing..."
+kind --version || (echo "Please install kind before running fvt tests"; exit 1)
+
+fvtlog "Creating Kubernetes cluster with kind"
+if [[ "$(kind get clusters -q | grep -c ${KIND_CLUSTER_NAME})" -ne 1 ]]; then
+  cat <<EOF | kind create cluster --name "${KIND_CLUSTER_NAME}" --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+EOF
+else
+  fvtlog "Warning! Running on existing cluster!"
+fi
+
+# Get KUBECONFIG
+kind get kubeconfig --name "${KIND_CLUSTER_NAME}" > "${SCRIPT_PATH}"/kubeconfig.yaml
+
+# Get all the worker nodes
+list_nodes
+
+fvtlog "Loading the staticrouter operator image to the cluster..."
+kind load docker-image --name="${KIND_CLUSTER_NAME}" "${REGISTRY_REPO}":"${CONTAINER_VERSION}"
+
+fvtlog "Apply common staticoperator related resources..."
+kubectl apply -f "${SCRIPT_PATH}"/../deploy/crds/iks.ibm.com_staticroutes_crd.yaml
```

Comment:
Maybe a for loop looks better here

Reply:
I've created an array which contains the resource files. Thank you!